server/devtool/src/main/java/org/eclipse/vorto/server/devtool/service/IInformationModelEditorService.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.vorto.server.devtool.service;
+
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.vorto.http.model.ModelId;
+import org.eclipse.vorto.http.model.ModelResource;
+
+/**
+ * 
+ * @author shiv
+ *
+ */
+public interface IInformationModelEditorService {
+
+	String linkFunctionBlockToInformationModel(String infoModelResourceId, ModelId functionBlockModelId, ResourceSet resourceSet,
+			Set<String> referencedResourceSet);
+
+	List<ModelResource> searchFunctionBlockByExpression(String expression);
```

Comment:
I can see that this search actually invokes a rest call to the Vorto Repository. Why? 
I would suggest that the frontend UI makes a REST call via angularJS directly to the Vorto Repository.